### DISCUSSION TITLE
Astroid menu options Badge Text

### DISCUSSION THREAD

**Discussion by ghost** [templaza/astroid-framework#394] [General]
Hello, I'm working on a website locally. I would like to change the badge text in the Astroid menu options. Larger font, bold and flashing font would be nice. However, I can't find the css class. Could someone tell me in which css file this is defined?
Many thanks in advance to kind regards
Olli

**Comment by Chacapamac** [comment]
@Solero-Artaxerxes 
Sorry nobody answer your question yet, Maybe you fix your problem?

Anyway to have more control over the Badge Text You can see what code build it and what HTML elements is building it by looking at your browser console.

![Screen Shot 2023-11-14 at 10 28 20 AM](https://github.com/templaza/astroid-framework/assets/1907276/74a17e6d-1fe9-480a-8b0e-39e9638229f9)

To CSS styling yo can use specific to a menu item or for the entire menu by using:

SPECIFIC
li.nav-item-id-169 span.menu-item-badge { whatever css here...)

or NON-SPECIFIC
li.nav-item span.menu-item-badge { whatever css here...)

**Comment by ghost** [comment]
@Chacapamac 

I thank them for their help, it helped me.

**Reply by Chacapamac** [reply]
span.menu-item-badge as selector
![Screen Shot 2024-01-15 at 8 52 21 AM](https://github.com/templaza/astroid-framework/assets/1907276/d5ef4208-dc8e-4d1e-b4ab-8b746d8b23a9)
